test/README.md

```diff
@@ -86,12 +86,40 @@ environment.
    For more info see
    [Velero Basic Install](https://velero.io/docs/v1.16/basic-install/)
 
-1. Install `helm` tool - on Fedora you can use:
+1. Install Helm (version ≥ 3.13 recommended).
 
+   You can install it in one of two ways:
+
+   **Option 1: Using DNF (recommended for recent Fedora/RHEL)**
+
+   ```bash
+   sudo dnf install -y helm
    ```
-   sudo dnf install helm
```

Comment:
We don't need to provide 2 options, this makes maintenance harder, and introduce issues with people running older distros or other disros (e.g. apt based). Best remove this text.